tests/unit/Core/Traits/MockPostData.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Tests\Unit\Core\Traits;
+
+
+
+class MockPostData
+{
+	use \Ushahidi\Core\Traits\StatefulData;
+    
+    protected $id;
+	protected $parent_id;
```

Comment:
Yeah there is something wrong with my editor, I started using visual studio because everyone recommended it but I'm not sure I like it that much, I find the split panes awkwardly sized :(